docs/docs_server_configuration_reference.md

```diff
@@ -305,3 +305,21 @@ how they should be spawned: in interior or exterior. By default all the npcs are
   // ...
 }
 ```
+
+## weaponStaminaModifiers
+
+This option allows you to flexibly adjust stamina forfeits of players' attacks using keywords set in Creation Kit.
+In case this field is not provided, some default, yet hardcoded, values are in use.
```

Comment:
Good description!

Please also let users know that it's (at this time) only active for servers with SweetPie.esp loaded.

Please also use grammarly or google translate to verify text quality

Reply:
Well, seems valid to me, but grammarly found a one little mistake, that is going to be fixed soon.

Reply:
ok